### ISSUE TITLE
Investigate adding conversion values to links on the units and conversion graphics

### ISSUE THREAD

**Issue by rchagolla** [OpenEnergyDashboard/OED#1386]
### Is your feature request related to a problem? Please describe.
Currently, the user cannot see the conversion values between units on the graphics page.

### Describe the solution you'd like

Investigate if adding conversion values to the links of the graphics is possible and if it is a beneficial addition to the graphic.

### Describe alternatives you've considered

If adding the conversion values is not beneficial, investigate if coloring links can be possible and beneficial to the graphic.

**Comment by NewtonLC** [comment]
Hello! Is [#1386](https://github.com/OpenEnergyDashboard/OED/issues/1386) available to work on? I would like to work on it if so!

**Comment by huss** [comment]
@NewtonLC Yes and I've assigned you the issue. Please let me know if you need anything.

**Comment by huss** [comment]
Given the move forward on time varying conversions, they will not have a single value to place on the graphic. While OED could show the non-time varying ones, this effort will wait to see how everything plays out to decide if and how it should move forward.

**Comment by huss** [comment]
I've decided that OED isn't going to do this unless a good proposal or request is made.
